cmd/ci-operator-prowgen/main.go

```diff
@@ -31,6 +31,13 @@ const (
 	sentryDsnMountPath  = "/etc/sentry-dsn"
 	sentryDsnSecretPath = "/etc/sentry-dsn/ci-operator"
 
+	apiciCIOperatorMountName       = "apici-ci-operator-credentials"
```

Comment:
If these are used once, we should just inline the literal string.

Reply:
Done.